cli.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package celvet
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	api "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	apiinstall "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/install"
+	apiv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	structuralschema "k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
+	"k8s.io/apimachinery/pkg/runtime"
+	runtimeserializer "k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+func Lint(args []string, outputWriter io.Writer) (int, error) {
+	if len(args) != 2 {
+		return 1, fmt.Errorf("usage: %s crd-file", args[0])
```

Comment:
Is there a reasonable expectation that any error code other than 1 will be returned from this function in the future? They're only applicable in very specific situations. If not, maybe just omit the exit code from the return and always treat an error as a exit code 1 error in the caller?

Reply:
I think the only other situation where it might be warranted to return a different non-zero error code would be for errors during things like file IO/etc. though at present we return 1 for those as well. I'll remove the error code from `Lint` and just return 1 from the CLI if any errors are seen.

Reply:
Sounds good. Keeping it simple sounds like the right thing do to for now.